appengine-deploy/README.md

```diff
@@ -0,0 +1,129 @@
+<!--
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# appengine-deploy
+
+This action deploys your source code to App Engine and makes the URL available to
+later build steps via outputs. This allows you to parameterize your App Engine
+deployments.
+
+## Prerequisites
+
+This action requires Google Cloud credentials that are authorized to deploy an
+App Engine Application. See the Authorization section below for more information.
+
+## Usage
+
+```yaml
+steps:
+- id: deploy
+  uses: GoogleCloudPlatform/github-actions/appengine-deploy@master
+  with:
+    project_id: ${{ secrets.PROJECT_ID }}
+
+# Example of using the output
+- id: test
+  run: curl "${{ steps.deploy.outputs.url }}"
+```
+
+## Inputs
+
+- `project_id`: (Required)
```

Comment:
When possible let's try to keep consistency with other actions documentation for common fields like this. See: https://github.com/GoogleCloudPlatform/github-actions/tree/master/setup-gcloud